.circleci/config.yml

```diff
@@ -1,17 +1,20 @@
 version: 2
 
-# NOTE: We run these in CircleCI because it has better artifacts support.
-# TODO: Move to GitHub Actions when its artifacts support improves.
+# NOTE: We run these in CircleCI because it has better artifacts support
+# and it is possible to publish image diffs as HTML like pytest-mpl in the future.
 jobs:
 
   image-tests-mpl302:
     docker:
-      - image: astropy/image-tests-py37-mpl302:1.10
+      - image: circleci/python:3.9
```

Comment:
For the Matplotlib 3.0.2 config try using Python 3.7 or 3.8 - I think currently you are trying to install numpy from source on Python 3.9 because there were no wheels. I think it'd be good to have a mix of Python versions for these tests anyway so could use 3.7 for 3.0.x, 3.8 for 3.1.x, and 3.9 for more recent versions?

Reply:
We're dropping Python 3.7, but I'll try 3.8. Thanks!

Do you know what's up with missing latex?

Reply:
Hmm. Same error with Python 3.8. That makes no sense!

Reply:
Oh, it is from scipy